examples/dummy-driver/src/main/java/org/apache/plc4x/java/examples/dummydriver/netty/DummyProtocol.java

```diff
@@ -18,26 +18,27 @@ Licensed to the Apache Software Foundation (ASF) under one
 */
 package org.apache.plc4x.java.examples.dummydriver.netty;
 
-import io.netty.buffer.ByteBuf;
-import io.netty.buffer.ByteBufUtil;
-import io.netty.buffer.Unpooled;
-import io.netty.channel.ChannelHandlerContext;
-import io.netty.handler.codec.MessageToMessageCodec;
+import java.util.List;
+
 import org.apache.plc4x.java.api.messages.PlcReadRequest;
 import org.apache.plc4x.java.api.messages.PlcRequest;
 import org.apache.plc4x.java.api.messages.PlcRequestContainer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.List;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufUtil;
+import io.netty.buffer.Unpooled;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.MessageToMessageCodec;
```

Comment:
we should look into why eclipse does sort the imports other than idea. We have a [.editorconfig](http://editorconfig.org/) (which eclipse doesn't do out-of-box and needs the plugin installed) but I don't think it respects import order. Without a unified import order we mess up our diff from time to time.

Reply:
Yeah, Eclipse's default import grouping config is java, javax, org, com, sorted within a group.  Number of imports needed to allow .* is 99.  Is that something that's configurable in editorconfig/IntelliJ?  If nothing else if there was a DEVELOPMENT.md page at least it could document what the rules are.

Reply:
It is still an open feature request: https://github.com/editorconfig/editorconfig/issues/231
We should define it therefore in the DEVELOPMENT.md.
Let's poll this on the mailinglist.